juju_verify/exceptions.py

```diff
@@ -24,3 +29,17 @@ class VerificationError(Exception):
 
 class CharmException(Exception):
     """Exception related to the charm or the unit that runs it."""
+
+
+class ActionFailed(Exception):
```

Comment:
I wonder if this class should be called `JujuActionFailed` or if it would seem it comes from the juju lib?